components/WithdrawLiquidity/WithdrawLiquidityController.tsx

```diff
@@ -17,6 +17,7 @@ const WithdrawLiquidity = () => {
   const {provider, accountInfo} = useContext(WalletContext)
   const [uniPrice, setUniPrice] = useState<string>('0')
   const [liquidityAmt, setLiquidityAmt] = useState<number>(0)
+  const [totalLiquidityAmt, settotalLiquidityAmt] = useState<string>('0')
```

Comment:
settotalLiquidityAmt -> change to setTotalLiquidityAmt

Reply:
Done